app/lib/pre_assembly/bundle.rb

```diff
@@ -134,7 +134,7 @@ def digital_object_base_params
     def discover_containers_via_manifest
       raise RuntimeError, ':manifest_cols must be specified' unless manifest_cols
       col_name = manifest_cols[:object_container].to_sym
-      raise RuntimeError, "object_container must be specified in manifest_cols: #{manifest_cols}" unless col_name
+      raise RuntimeError, "object must be specified in manifest_cols: #{manifest_cols}" unless col_name
```

Comment:
This seems backwards to me.  It literally is looking for `[:object_container]`, so isn't that what the error should indicate?

Reply:
it's looking for a column named "object".  I figured this error should be about the column name, not our variable name.

Reply:
Ah, I see.  We've hardcoded `manifest_cols[:object_container] = 'object'`